syntax = "proto3";

message CompressedMessage {
  repeated int32 struct_id = 1;
  bytes message = 2;
  repeated int32 message_sizes = 3;

}